kafka-ui-react-app/src/components/Brokers/Broker/Configs/InputCell.tsx

```diff
@@ -71,14 +81,15 @@ const InputCell: React.FC<InputCellProps> = ({ row, getValue, onUpdate }) => {
       }
     >
       <S.Value title={initialValue}>{initialValue}</S.Value>
-      <Button
+      <ActionButton
         buttonType="primary"
         buttonSize="S"
         aria-label="editAction"
         onClick={() => setIsEdit(true)}
+        canDoAction={canEditBrokersConfig}
       >
         <EditIcon /> Edit
-      </Button>
+      </ActionButton>
```

Comment:
Can the action button do the same check inside? So we can send what permissions we need and button will decide

Reply:
I was actually thinking about this , but then i thought about testing that is why i was hesitant.

cause the user we see that his button is disabled , but then he does not see the hook. even though he has to mock it , to make it active.

but if that is okay i can transform it.

Reply:
From my point of view we need to test it only once: test how this action button works with different set of props. 
for other test cases we can stub the whole button

Reply:
yep sounds good , i will change those action components, i will remove permission based test from the pages and add tests in those actions components.

Reply:
Implemented.